tests/transports-governance/test_runner_hierarchical_comprehensive.py

```
@@ -0,0 +1,830 @@
+class ComprehensiveTestRunner:
+    """Main test runner for all governance test suites"""
+
+    # Test suites in execution order
+    TEST_SUITES = [
+        "test_governance_hierarchical_models",
+        "test_governance_hierarchical_validation",
+        "test_governance_usage_tracking_hierarchical",
+        "test_governance_api_endpoints_comprehensive",
+        "test_governance_integration_flow_hierarchical",
+        "test_governance_edge_cases_comprehensive",
+    ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)